koap/src/commonMain/kotlin/Header.kt

```diff
@@ -1,7 +1,9 @@
+@file:JsExport
```

Comment:
ultra ultra ultra nitpick: newline after

```suggestion
@file:JsExport

```

Reply:
I originally had a newline but the IDE complained 🤪

Reply:
Weird. No newline is fine if it's absence is needed to quiet the IDE.

Reply:
Super weird. And since merging latest `main` it seems happy with a newline now.